fluss-lake/fluss-lake-paimon/src/test/java/org/apache/fluss/lake/paimon/LakeEnabledTableCreateITCase.java

```diff
@@ -474,16 +732,8 @@ void testAlterLakeEnabledLogTable() throws Exception {
 
         // try to enable lake table again
```

Comment:
Enhance the test with the most common case:
1: write some data
2: disable datalake 
3: enable datalake again, verify data still exist. we can just verify snapshot for clean code.